File: codes/java/practice/baiscs/pattern/pattern3.java
import java.util.Scanner;
public class pattern3{
	public static void main(String[] args){
		Scanner cin=new Scanner(System.in);
		int n=cin.nextInt();
		// row
		int row=1;
		int nst =n;
		while(row<=n){
			// work
			int cst=1;
			while(cst<=nst){
				System.out.print("* ");
				cst++;
			}
			//prep
			System.out.println("");
			nst--;
			row++;
		}
	}
}

File: codes/cpp/basics/loops/pattern3.cpp
#include<iostream>
int main(){
	int n;
	std::cin>>n;
	for(int i=1;i<=n;i++){
		for(int j=1;j<=i;j++){
			std::cout<<(char)(64+j);
		}
		std::cout<<std::endl;
	}
	
}
